tools/console/console.js

```diff
@@ -309,7 +309,7 @@ var ProgressDisplayFull = function (console) {
   self._progressBarRenderer = new ProgressBarRenderer(PROGRESS_BAR_FORMAT, options);
   self._progressBarRenderer.start = new Date();
 
-  self._headless = false;
+  self._headless = !!(process.env.METEOR_HEADLESS && process.env.METEOR_HEADLESS != '0');
```

Comment:
I reused the same approach from this same file: https://github.com/mitar/meteor/blob/3ee60f57e57b56b56b36e8c8cdbdbb79c9b55e25/tools/console/console.js#L74

I wanted to be consistent inside a file.